OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVREntryPoint.cpp

```diff
@@ -29,13 +29,33 @@ DEFINE_LOG_CATEGORY(OSVREntryPointLog);
 
 OSVREntryPoint::OSVREntryPoint()
 {
+}
+
+void OSVREntryPoint::Initialize()
+{
+    // avoid double initialization
+    if (osvrClientContext != nullptr)
+    {
+        // too spammy to log here
+        return;
+    }
+
+    // hack: to avoid delay on OSVR server connection attempt when not using OSVR, don't initialize if nohmd command line param specified
+    if (FParse::Param(FCommandLine::Get(), TEXT("nohmd")))

```

Comment:
"Can this skipping behavior be parameterized?"

In what way were you thinking?   I think it best to make it skip by default with nohmd option to avoid the excessive startup pause for non-VR users, so perhaps add a parameter check that overrides the nohmd skip like "-forceosvr", so "-nohmd -forceosvr" on command line would still initialize OSVREntryPoint?